app/src/main/java/ar/scacchipa/twittercloneapp/viewmodel/SplashViewModel.kt

```diff
@@ -10,17 +10,17 @@ import kotlinx.coroutines.launch
 class SplashViewModel(
```

Comment:
why the changes in the splash? this was already closed. Also we miss the details that you were using mutable instead of liveData

Reply:
Because I didn't want to explose a MutableLiveData. It was fixed._